### ISSUE TITLE
Accessibility: Audit and improve accessibility of main/logo link 

### ISSUE THREAD

**Issue by derekshirk** [derekshirk/personal-site-2020#188]
> Links do not have a discernible name
>
> Link text (and alternate text for images, when used as links) that is discernible, unique, and focusable improves the navigation experience for screen reader users. Learn more.
> 
> Failing Elements
>
> <a class="Logo u-flex u-borderNone" href="/">
